nyxx/lib/src/utils/CacheUtility.dart

```diff
@@ -0,0 +1,113 @@
+part of nyxx;
+
+/// Cache Utility is a Utility that lets you retrieve Entities (from the cache) from outside of the nyxx project. Typically used in nyxx.* projects.
+/// An example getting a user is below:
+/// ```dart
+/// void main() {
+///     var bot = Nyxx("TOKEN");
+///     Cacheable<Snowflake, User> cachedUser = CacheUtility.users(bot, Snowflake(''));
+/// }
+/// ```
+class CacheUtility {
+  /// Retrieves a cached User.
+  /// ```dart
+  /// void main() {
+  ///     var bot = Nyxx("TOKEN");
+  ///     Cacheable<Snowflake, User> cachedUser = CacheUtility.users(bot, Snowflake(''));
+  /// }
+  /// ```
+  Cacheable<Snowflake, User> users(
+    Nyxx client,
+    Snowflake id,
+  ) =>
```

Comment:
Keep these in one line, thats too verbose for me and less readable. Use 120 charavter line width please

Reply:
Fixed in next Push, removed trailing commas also.